app/views/IndexView.scala.html

```diff
@@ -20,120 +20,142 @@
 @import viewmodels._
 @import utils.CurrencyFormatter.currencyFormat
 @import java.time.format.DateTimeFormatter
+@import java.time.LocalDate
+@import models.exclusions.ExcludedTrader
 
 @this(
-        layout: templates.Layout,
-        govukButton: GovukButton
+layout: templates.Layout,
+govukButton: GovukButton
 )
 @(
-        businessName: String,
-        vrn: String,
-        returnsViewModel: ReturnsViewModel,
-        paymentsViewModel: PaymentsViewModel,
-        paymentError: Boolean
+businessName: String,
+vrn: String,
+returnsViewModel: ReturnsViewModel,
+paymentsViewModel: PaymentsViewModel,
+paymentError: Boolean,
+excludedTrader: Option[ExcludedTrader]
 )(implicit request: Request[_], messages: Messages)
 
 @layout(
-    pageTitle = titleNoForm(messages("index.title")),
-    showBackLink = false,
-    fullWidth = true
+pageTitle = titleNoForm(messages("index.title")),
+showBackLink = false,
+fullWidth = true
```

Comment:
Most of this file has gone skewiff with the indentation

Reply:
code formatted